cf-agent/verify_files.c

```diff
@@ -575,15 +577,16 @@ static bool SaveBufferCallback(const char *dest_filename, void *param, NewLineMo
     return true;
 }
 
+
 static PromiseResult RenderTemplateMustache(EvalContext *ctx, const Promise *pp, Attributes a,
-                                            EditContext *edcontext)
+                                            EditContext *edcontext, char *template)
 {
     PromiseResult result = PROMISE_RESULT_NOOP;
+    JsonElement *default_template_data = NULL;
```

Comment:
I might be missing something, but is `default_template_data` used anywhere?

Reply:
It was used to destroy the created JSON struct. This PR introduced a memory leak by removing the appropriate `JsonDestroy`. Fixed in https://github.com/cfengine/core/pull/3457

Reply:
+1 good find. Also +1 for activating valgrind for unit and acceptance tests. :-)